test/blackbox-tests/test-cases/pkg/dependency-install-file.t

```diff
@@ -57,4 +57,12 @@ With this project set up, lets depend on it.
   $ cat > foo.ml <<EOF
   > let () = Nondune.main ()
   > EOF
-  $ dune exec ./foo.exe 2>&1 | sed -e 's#File unavailable: .*ocaml/nondune.cma$#File unavailable: ocaml/nondune.cma#'
+  $ dune exec ./foo.exe 2>&1 | dune_cmd stdlib-path
```

Comment:
I think that you can:
- get that from `ocamlc -config`
- use `BUILD_PATH_PREFIX_MAP`

```
  $ OCAML_STDLIB=$(ocamlc -config|grep standard_library:|cut -d ' ' -f2)
  $ BUILD_PATH_PREFIX_MAP="/OCAML_STDLIB=$OCAML_STDLIB:$BUILD_PATH_PREFIX_MAP"
```